### PR TITLE
Update geometry-schema.json

### PR DESCRIPTION
**Pull request by juliozinga**
Adjust array in definitions/position

### CODE REVIEW

File: geometry-schema.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
             "description": "A single position",
             "type": "array",
             "minItems": 2,
-            "items": [ { "type": "number" }, { "type": "number" } ],
+            "items": { "type": "number" },
```
[the remark this answers is not in the document]
Comment (juliozinga) [reply geometry-schema.json]: In RFC 7946 -  The GeoJSON Format defines position may has a third number representing altitude.

 A position is an array of numbers. There MUST be two or more elements. The first two elements are longitude and latitude, or easting and northing, precisely in that order and using decimal numbers. Altitude or elevation MAY be included as an optional third element.

Then, maxItems equals to three.

**Comment by chicco785** [comment]
My recommendation is to reference the new official schema: http://geojson.org/schema/Geometry.json


Dr. FEDERICO MICHELE FACCA
CTO
0041 78 807 58 38
Martel Innovate  -  Professional support for innovation projects 	
Click to download our innovators' insights! 	
Follow Us on Twitter

File: geometry-schema.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (ldiez) [reply geometry-schema.json]: Actually, as it is defined, it does not admit more than 2 elements in the array.

The definition of items is more restrictive than minItems, somehow it equals maxItems to 2. The proposed change would fix it.

**Comment by jmcanterafonseca** [comment]
as pointed out by @chicco785 we are going to use the new schema. PR #346, thus closing this one

Thanks!

**Pull request closed without merging**